lib/foods.js

```diff
@@ -78,14 +80,24 @@ function addFood() {
   validateInputs();
 
   if ( $('#name-field input').val() != '' && $('#calories-field input').val() != '') {
-    var food = $('#name-field input').val();
+    var name = $('#name-field input').val();
     var calories = $('#calories-field input').val();
-    var foodsRaw = localStorage.getItem("foods");
-    var foods = $.parseJSON(foodsRaw);
-    foodTableSingleRow($('#food-list'), food, calories, true, false, true);
-    saveFoodList($('#food-list'));
+    var foodsRaw = localStorage.getItem("all-foods");
+    var allFoods = $.parseJSON(foodsRaw);
+    if (allFoods == null) {
+      allFoods = [];
+    }
     clearInputs();
+    var newFood = createNewFood(name, calories);
+    allFoods.push(newFood);
+    sortedFoods = allFoods.sort(function (a,b) {
+      return b.id - a.id;
+    });
+    console.log(sortedFoods);
```

Comment:
@lucyconklin Eventually we will have to get rid of `console.log` lines. During final refactor, we should make a point to do a general search for it.